pkgs/build-support/rust/fetch-cargo-vendor-util.py

```diff
@@ -205,13 +205,14 @@ def extract_crate_tarball_contents(tarball_path: Path, crate_out_dir: Path) -> N
 def create_vendor(vendor_staging_dir: Path, out_dir: Path) -> None:
     lockfile_path = vendor_staging_dir / "Cargo.lock"
     out_dir.mkdir(exist_ok=True)
+    (out_dir / "registory").mkdir(exist_ok=True)
```

Comment:
```suggestion
    (out_dir / "registry").mkdir(exist_ok=True)
```

Please correct this everywhere

Reply:
Sorry. I fixed it now.